assets/Changelog.md

```diff
@@ -21,6 +21,7 @@ Added search filter: `IncludeClassNames`. ([UE4SS #472](https://github.com/UE4SS
 ### UHT Dumper
 
 ### Lua API
+Added global function `CreateBlankObject`, which returns an empty UObject. ([UE4SS #652](https://github.com/UE4SS-RE/RE-UE4SS/issues/652)) 
```

Comment:
```suggestion
Added global function `CreateBlankObject`, which returns an invalid UObject. ([UE4SS #652](https://github.com/UE4SS-RE/RE-UE4SS/issues/652)) 
```
I think this makes more sense.
I don't think "empty" makes it clear what the purpose of this function is.